packages/perseus/src/widgets/grapher/grapher.tsx

```diff
@@ -551,14 +551,14 @@ class Grapher extends React.Component<Props> implements Widget {
     render(): React.ReactNode {
         const type = this.props.plot.type;
         const coords = this.props.plot.coords;
-        const asymptote = this.props.plot.asymptote;
+        const asymptote = (this.props.plot as any).asymptote;
```

Comment:
This is because of some weird union type stuff in TS. Some plot types have `asymptote` and some don't.

Reply:
You could instead do something like the below to keep some measure of typing, but it is more complex than just casting, so maybe it's not as easy to understand.
`const asymptote = "asymptote" in this.props.plot ? this.props.plot.asymptote : undefined;`